src/index.js

```diff
@@ -17,25 +22,67 @@ export default class Differencify {
   constructor(conf) {
     this.configuration = sanitiseGlobalConfiguration(conf);
     this.ChromyRunner = new ChromyRunner(this.configuration);
+    this.chromeInstances = {};
+    this.chromeInstancesId = CHROME_PORT;
     if (this.configuration.debug === true) {
       logger.enable();
     }
     createDir(this.configuration.screenshots);
     createDir(this.configuration.testReportPath);
   }
+  _createChromeInstance(testConfig) {
+    const width = testConfig.resolution.width || CHROME_WIDTH;
+    const height = testConfig.resolution.height || CHROME_HEIGHT;
+    const flags = [`--window-size=${width},${height}`];
+    const chromy = new Chromy({
+      chromeFlags: flags,
+      port: this.chromeInstancesId,
+      waitTimeout: this.configuration.timeout,
+      visible: this.configuration.visible || false,
```

Comment:
This option already defaults to `false` when the configs get merged in `sanitiseGlobalConfiguration`, no need to set again.